nisystemlink/clients/auth/models/_additional_models.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import Dict, List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+from ._auth_models import AuthPolicy, Error, Org, User, Workspace
+
+
+class AuthResponse(JsonModel):
```

Comment:
My preference is to avoid using the 'Response' suffix on models. There's a few arguments against it, such as:
- It would be redundant to put it on everything
- Doesn't convey any useful meaning to users

Couple ideas are `AuthResult` or `AuthInfo`, but I'm open to alternatives.

Reply:
The term 'Response' was used to be consistent with the OpenAPI yaml. I am good with changing it to 'AuthInfo' for this instance.
![image](https://github.com/ni/nisystemlink-clients-python/assets/30560648/7b99f0c7-5570-4dfe-8611-8facd77d629c)